src/quo2/foundations/colors.cljs

```diff
@@ -185,95 +186,130 @@
 (def info-50-opa-30 (alpha info-50 0.3))
 (def info-50-opa-40 (alpha info-50 0.4))
 
-;; Customization
-(def purple-20 (alpha "#8661C1" 0.2))
-(def purple-50 "#8661C1")
-(def purple-60 (alpha "#8661C1" 0.6))
-(def indigo-20 (alpha "#496289" 0.2))
-(def indigo-50 "#496289")
-(def indigo-60 (alpha "#496289" 0.6))
-(def turquoise-20 (alpha "#448EA2" 0.2))
-(def turquoise-50 "#448EA2")
-(def turquoise-60 (alpha "#448EA2" 0.6))
```

Comment:
These values are wrong, (ex. `:purple-20` represents different color, not opacity)